search/tasks.py

```diff
@@ -44,6 +43,10 @@ def wrap_retry_exception(*exception_classes):
     try:
         yield
     except Exception as ex:
+        if isinstance(ex, NotFound):
+            # No corresponding reddit post/comment found for django model object.  Log error and continue indexing.
+            log.exception(str(ex))
```

Comment:
You can omit the `str(ex)` here, the whole stack trace will be logged automatically

Reply:
✔️